components/ProjectDetail/index.js

```diff
@@ -1,25 +1,23 @@
 import Heading from "../Heading";
-import { ColumnSection } from "../../styles";
+import { ColoredFont, HeavyFont, LightFont } from "../../styles";
 import { RowSection } from "../../styles";
 import Image from "next/image";
 import BackIcon from "../Icon/BackIcon";
 import { useRouter } from "next/router";
 import StyledButton from "../StyledButton";
 import ProjectForm from "../ProjectForm";
 import { useState } from "react";
-import dayjs from "dayjs";
 import useSWR from "swr";
 import { Main } from "../../styles";
+import styled from "styled-components";
+import { handleProjectRestructure } from "../handelProjectResructure";
+import { ImageWrapper } from "../../styles";
+import ConfirmDeleteProject from "../ComfirmDeleteProject";
```

Comment:
You could refactor also this import section multiple imports from similar sources just like in `line 2` there's 3 more for example in lines `3`, `11`, and `14`
I'm for one of the following 2 designs:
1. Spread imports to have one imported package per line
2. Keep imports tight to as few lines as possible
but not mixed 😊